packages/conform-zod/index.ts

```diff
@@ -96,36 +96,37 @@ export function getFieldsetConstraint<Source extends z.ZodTypeAny>(
 		'pattern',
 	];
 
-	function resolveFieldsetConstarint<T extends Record<string, any>>(
+	function resolveFieldsetConstraint<T extends Record<string, unknown>>(
 		schema: z.ZodType<T>,
 	): FieldsetConstraint<z.input<Source>> {
 		if (schema instanceof z.ZodObject) {
-			const result: FieldsetConstraint<z.input<Source>> = {};
+			const result: Record<string, FieldConstraint<unknown>> = {};
```

Comment:
the idea here is to cast it to a type that is currently known within this code block